torch/distributed/distributed_c10d.py

```diff
@@ -987,6 +987,11 @@ def _is_barrier_after_init() -> int:
     return int(os.getenv("TORCH_DIST_INIT_BARRIER", "0"))
 
 
+def _abort_in_destroy_pg() -> bool:
+    # Environment variable to control whether to abort the communicators when users call destroy_process_group()
+    return os.getenv("TORCH_NCCL_ABORT_IN_DESTROY_PG") is not None
```

Comment:
Need to be stricter by checking the value, will update in  the next version:
`os.getenv("TORCH_NCCL_ABORT_IN_DESTROY_PG") == "1" or os.getenv("TORCH_NCCL_ABORT_IN_DESTROY_PG").lower() == "true"`

Reply:
yea, if we have the JK thing hooked up, the value will always be set by JK so this will be badly broken